ui/src/app/pages/profile/data-user-code-of-conduct.tsx

```diff
@@ -169,12 +169,11 @@ export const DataUserCodeOfConduct = withUserProfile()(
                 </div>
                 <InitialsAgreement onChange={(v) => this.setState({initialMonitoring: v})}>
                   My work, including any external data, files, or software I upload into the
-                   Researcher Workbench, may be logged and monitored by the
-                   <i>All of Us</i> Research Program to ensure compliance with policies and
-                   procedures.
+                   Researcher Workbench, will be logged and monitored by the <i>All of Us</i>
+                   Research Program to ensure compliance with policies and procedures.
```

Comment:
Need a space between All of Us and Research Program. (screenshot shows 'All of UsResearch Program')

Reply:
dangit. react doesn't like the line break around the i tag.